packages/actionmenu/jest.config.js

```diff
@@ -1,9 +1,11 @@
+const path = require('path')
+
 const baseConfig = require('../../jest/base.config.js')
-const { name } = require('./package')
+const { name } = require('./package.json')
 
 module.exports = {
   ...baseConfig,
   displayName: name,
   name: name,
-  testMatch: [`${__dirname}/**/*/?(*.)+(spec|test).(ts|tsx)`]
+  testMatch: [path.join(__dirname, '/**/*/?(*.)+(spec|test).(js|ts|tsx)')]
```

Comment:
really not a great reason to *partially* use path.join.  It'll be OS-specific only for that first separator.  But we still have forward-slashes hardcode after that.

Reply:
yeah, it's some new linting rule. i think it's when used with `__dirname` 🤷